### ISSUE TITLE
when selecting final favorite of each type at the bottom across all regions, can dual types be considered in multiple columns?

### ISSUE THREAD

**Issue by gutpunch333** [CajunAvenger/cajunavenger.github.io#130]
For example, my favorite ground type from gen 8 is ursaluna, and my favorite normal type from gen 8 is wyrdeer. My favorite ground type of all time is excadrill, so in the bottom row it takes precedence over ursaluna. my favorite normal type of all time however, i like ursaluna better than all of the options i have for normal types, but I can't put ursaluna there even though it's a favorite and a normal type, because it only shows up in the ground category. 

the reason I don't have ursaluna as my favorite normal type of gen 8 is because I like wyrdeer better than all of the ground types besides ursaluna. if i put ursaluna for the normal type over wydeer, I would have to pick a less desirable ground type to be my favorite for gen 8 and wyrdeer wouldn't be on the list. 

tl:dr make all time favorites in bottom row fully consider dual-typed pokemon

**Comment by CajunAvenger** [comment]
you can put Ursaluna in the Generation slot, pick it for the bottom row, and then switch the generation slot to Wyrdeer. Ursaluna will still be in the bottom row until you choose a different one
